cypress/e2e/data-import/uploading-files/upload-2-files-same-file-extensions.cy.js

```diff
@@ -0,0 +1,43 @@
+import { DevTeams, TestTypes, Permissions } from '../../../support/dictionary';
+import Users from '../../../support/fragments/users/users';
+import TopMenu from '../../../support/fragments/topMenu';
+import DataImport from '../../../support/fragments/data_import/dataImport';
+import JobProfiles from '../../../support/fragments/data_import/job_profiles/jobProfiles';
+import JobProfileView from '../../../support/fragments/data_import/job_profiles/jobProfileView';
+
+describe('data-import', () => {
+  describe('Uploading files', () => {
+    let user;
+
+    before('create test data', () => {
+      cy.createTempUser([Permissions.moduleDataImportEnabled.gui]).then((userProperties) => {
+        user = userProperties;
+        cy.login(user.username, user.password, {
+          path: TopMenu.dataImportPath,
+          waiter: DataImport.waitLoading,
+        });
+      });
+    });
+
+    after('delete test data', () => {
+      Users.deleteViaApi(user.userId);
+    });
+
+    it(
+      'C2378 UIDATIMP-252: Uploading 2 files with the same extension, but different case, does not work (folijet) (TaaS)',
+      { tags: [TestTypes.extendedPath, DevTeams.folijet] },
+      () => {
+        const filesNames = ['C2378_File1.mrc', 'C2378_File2.MRC'];
+        const jobProfileToRun = 'Default - Create instance and SRS MARC Bib';
+        DataImport.verifyUploadState();
+        DataImport.uploadBunchOfDifferentFiles(filesNames);
+        JobProfiles.waitLoadingList();
+        JobProfiles.search(jobProfileToRun);
+        JobProfiles.runImportFile();
+        JobProfiles.waitFileIsImported(filesNames[0]);
+        JobProfileView.verifyInstanceImportStatus(0, 1, 'Completed');
```

Comment:
We have the function for checking status Logs.checkStatusOfJobProfile(JOB_STATUS_NAMES.COMPLETED);

Reply:
But i need to check both first and second row , what you suggest with that context how use this function Logs.checkStatusOfJobProfile(JOB_STATUS_NAMES.COMPLETED); ?

Reply:
You can modify the function like
checkStatusOfJobProfile: (status = 'Completed', rowNumber = 0) => cy.do(MultiColumnListCell({ row: rowNumber, content: status }).exists())
and use it like
[0,1].forEach(rownumber => {
Logs.checkStatusOfJobProfile(JOB_STATUS_NAMES.COMPLETED, rowNumber);
});

Reply:
I know that i can modify , but i do not think it's necessary

Reply:
If we have the function why we should create another one?

Reply:
But we must create function 
Because it's not good idea use function .then. forEach and etc. in cy.js (spec)file
anyway we need to write this type of things in fragment . For that i creat new function